pd-client/client.go

```diff
@@ -469,8 +483,13 @@ func (c *client) GetTS(ctx context.Context) (int64, int64, error) {
 }
 
 func (c *client) GetRegion(ctx context.Context, key []byte) (*metapb.Region, *metapb.Peer, error) {
+	if span := opentracing.SpanFromContext(ctx); span != nil {
+		span = opentracing.StartSpan("pdclient.GetRegion", opentracing.ChildOf(span.Context()))
+		defer span.Finish()
```

Comment:
I prefer not using defer in if closure.

Reply:
I'm not quite understand... It's not in a closure here, just a normal if statement.
The check says: if the span information in the context is missing, don't trace this query.

Reply:
the defer will be called after the function returns. 
But sometimes I think this will be called after if, so we should make the defer more scene.

Reply:
So, what do you suggest to write here?

Reply:
If the condition does not satisfy, the `defer` will not execute, and the `opentracing.StartSpan` will not run, neither.

Reply:
no better idea now